### ISSUE TITLE
this is a cookielogger dumbasses

### ISSUE THREAD

**Issue by 19clee** [OriginalAlien/Roblox-Mass-Report#24]
if you legit looked in the code then you would see it logs your cookie dumbfucks its also python so if it doesnt work then it means you dont have the right libs

**Comment by sarahmae356** [comment]
yeah it gets your cookie, from "cookies.txt" where your supposed to put your cookie, that how it makes the reports, if it didn't use the cookie it would be like trying to report someone while your logged out. it also doesn't do anything with the cookies provided to it other then send requests to the Roblox platform, if it was a cookie logger you would likely see a discord webhook link in the code. your also wrong about the libs too, if it doesn't work its because it hasn't worked for a while because 1, the page "https://www.roblox.com/build/upload" doesn't exist anymore which it uses to get the verification token and 2, Roblox doesn't use a verification token for anything anymore.

**Comment by ArchivesXen0** [comment]
@19clee  are you stupid?

**Comment by robloxuserig** [comment]
@19clee youre a dumbass

**Comment by silas-vale** [comment]
@19clee Do you even know codes also if you check the fucking cookies.txt it's empty albert and einstein so how is it suppose to work you gotta manually put the cookies in
